api/controllers/task.go

```diff
@@ -78,3 +79,8 @@ func (t *TaskController) ListTasks(c *fireball.Context) (fireball.Response, erro
 	return fireball.NewJSONResponse(200, summaries)
 
 }
+
+func (t *TaskController) UpdateTask(c *fireball.Context) (fireball.Response, error) {
```

Comment:
There is no `UpdateTask`

Reply:
I have removed the implementation, including the route at the top of the file